### DISCUSSION TITLE
[Already implemented/Solved] Feature request: Minify new unsaved file

### DISCUSSION THREAD

**Discussion by uriasbt** [Josee9988/MinifyAll#110] [Ideas]
This extension uses the file type to figure how to minify the content, so if I have a new unsaved file with some Json content it does not work until I save the file as JSON.
It would be awesome if i could simply select a option "minify selected text as JSON", just like the beautify extension I use: https://github.com/HookyQR/VSCodeBeautify

**Comment by Josee9988** [comment]
You can select the language type without saving, later I can record the screen showing you that.
Thanks for the idea!

**Comment by Josee9988** [comment]
@uriasbt In this video y minified CSS and JSON in a scratch file (without actually saving the file)
You can also do it with all the other available languages.
Just select the language you are using and minify :)

https://user-images.githubusercontent.com/38838976/119021399-adb21580-b99f-11eb-8e4f-ca0c70b64fc6.mp4
